src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3390,39 +3551,130 @@ msgstr "Haal Centraal API"
 msgid "Haal Centraal"
 msgstr "Haal Centraal"
 
+#: openforms/prefill/contrib/haalcentraal/plugin.py:115
+#: openforms/prefill/contrib/stufbg/plugin.py:134
+msgid "Service not selected"
+msgstr "Service is niet geselecteerd."
+
+#: openforms/prefill/contrib/haalcentraal/plugin.py:129
+#: openforms/prefill/contrib/kvk/plugin.py:94
+#: openforms/prefill/contrib/stufbg/plugin.py:143
+#: openforms/registrations/contrib/objects_api/checks.py:38
+#, python-brace-format
+msgid "Client error: {exception}"
+msgstr "Client fout: {exception}"
+
+#: openforms/prefill/contrib/haalcentraal/plugin.py:136
+msgid "Missing status 404 in response body"
+msgstr "Geen 404 in antwoord bericht gevonden."
+
+#: openforms/prefill/contrib/haalcentraal/plugin.py:141
+msgid "Check call unexpectedly succeeded for test BSN '{}'"
+msgstr ""
+"Controle verzoek geeft onverwacht een resultaat voor een test BSN: '{}'"
+
 #: openforms/prefill/contrib/kvk/apps.py:8
 msgid "KvK prefill plugin"
 msgstr "KvK prefill plugin"
 
-#: openforms/prefill/contrib/kvk/plugin.py:23
+#: openforms/prefill/contrib/kvk/plugin.py:26
 msgid "KvK Company by KvK number"
 msgstr "KvK Bedrijf via KvK-nummer"
 
+#: openforms/prefill/contrib/kvk/plugin.py:89
+#, python-brace-format
+msgid "Configuration error: {exception}"
+msgstr "Configuratiefout: {exception}"
+
+#: openforms/prefill/contrib/kvk/plugin.py:101
+msgid "Response not a dictionary"
+msgstr "Antwoord is geen object"
+
+#: openforms/prefill/contrib/kvk/plugin.py:104
+msgid "Response does not contain results"
+msgstr "Antwoord bevat geen resultaten"
+
+#: openforms/prefill/contrib/kvk/plugin.py:108
+#, python-brace-format
+msgid "Did not find kvkNummer='{kvk}' in results"
+msgstr "Kon KvK nummer {kvk} niet vinden in de resultaten."
+
 #: openforms/prefill/contrib/stufbg/apps.py:8
 msgid "StUF-BG prefill plugin"
 msgstr "StUF-BG prefill plugin"
 
-#: openforms/prefill/contrib/stufbg/plugin.py:38
+#: openforms/prefill/contrib/stufbg/plugin.py:43
 msgid "StUF-BG"
 msgstr "StUF-BG"
 
+#: openforms/prefill/contrib/stufbg/plugin.py:150
+#, python-brace-format
+msgid "SyntaxError in response: {exception}"
+msgstr "Structuurfout in antwoord: {exception}"
+
+#: openforms/prefill/contrib/stufbg/plugin.py:157 stuf/stuf_zds/client.py:427
+#, python-brace-format
+msgid "Unexpected response: expected '{message}' SOAP response"
+msgstr "Onverwacht antwoord: Verwacht '{message}' SOAP antwoord"
+
+#: openforms/prefill/models.py:13
+msgid "default person plugin"
+msgstr "standaard persoonsinformatie plugin"
+
+#: openforms/prefill/models.py:15
+msgid "Default prefill plugin to use for person data"
+msgstr "Standaard prefill-plug-in om te gebruiken voor persoonsgegevens"
+
+#: openforms/prefill/models.py:18
+msgid "default company plugin"
+msgstr "standaard bedrijfsinformatie plugin"
+
+#: openforms/prefill/models.py:20
+msgid "Default prefill plugin to use for company data"
+msgstr "Standaard prefill-plug-in om te gebruiken voor bedrijfsgegevens"
```

Comment:
```suggestion
msgstr "Standaard prefill-plugin om te gebruiken voor bedrijfsgegevens"
```

Reply:
Ja eh, eerder verbeterde je "prefill plugin" naar prefillplugin". Wat is het nou!?